docs/content/page/index.en-us.md

```diff
@@ -11,22 +11,21 @@ draft: false
 # What is Gitea?
 
 Gitea is a painless self-hosted Git service. It is similar to GitHub, Bitbucket, and GitLab.
-Gitea is a fork of [Gogs](http://gogs.io). See the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
+Gitea is a fork of repository named [Gogs](http://gogs.io). Kindly check out the [Gitea Announcement](https://blog.gitea.io/2016/12/welcome-to-gitea/)
 blog post to read about the justification for a fork.
 
 ## Purpose
 
 The goal of this project is to provide the easiest, fastest, and most painless way of setting
 up a self-hosted Git service.
 
-With Go, this can be done platform-independently across
-**all platforms** which Go supports, including Linux, macOS, and Windows,
+This is possible with Go, it is platform-independent and supports **all platforms**, including Linux, macOS, and Windows,
```

Comment:
This is different from the previous statement. Gitea should work on all platforms that Go supports, but Go doesn't support "all" platforms.

Reply:
Indeed. Go for example does not support Solaris 😉

Reply:
> Indeed. Go for example does not support Solaris 😉

It should support Solaris on amd64 arch. 
https://go.dev/doc/install/source#environment

Reply:
I mean Solaris on Spark 😆